pkg/keystore/keystore.go

```diff
@@ -93,6 +95,21 @@ func (k *keystore) CreateAndExportKey(kt kms.KeyType) (string, []byte, error) {
 	return keyID, b, nil
 }
 
+// ImportKey imports private key bytes (in DER format) of kt type into KMS and returns key ID.
+func (k *keystore) ImportKey(der []byte, kt kms.KeyType) (string, error) {
+	pk, err := x509.ParsePKCS8PrivateKey(der)
+	if err != nil {
+		return "", fmt.Errorf("import key: %w", err)
+	}
```

Comment:
Assuming keys to be imported are in ASN.1 DER format is incorrect. The kt must be checked first, `ecdsa` keys can be in either DER or IEEEP_P1363. Also JWS's only supported format is IEEE_P1363 ([JWS apparently doesn't support DER format](https://github.com/hyperledger/aries-framework-go/blob/main/pkg/kms/localkms/localkms.go#L197)).

For now, we can assume that webkms will receive the private key in PKCS8 (ASN.1 DER) format only. It then internally can generate either a DER or an IEEE_P1363 key (as long as `x509.ParsePKCS8PrivateKey(der)` returns `*ecdsa.PrivateKey`).

for Ed25519 keys, simply do `ed25519.PrivateKey(keyBytes)`

we will soon support BBS+ keys as well, this will be another type.

please add kt switch conditions here to set the right `pk` type

Reply:
please disregard my comment about :
> for Ed25519 keys, simply do ed25519.PrivateKey(keyBytes)

I noticed that `x509.ParsePKCS8PrivateKey` can parse ed25519 private keys as well.

but we still need a kt switch condition for key types that can use `x509.ParsePKCS8PrivateKey`.

Reply:
@aholovko @Baha-sk Can we have that as follow up PR to this PR? is this kt switch condition is going to block SAVE ANY DID module in edge-agent?

Can we get this PR merged to unblock ourselves for now.

Reply:
@Baha added explicit list of supported key types

Reply:
> @aholovko @Baha-sk Can we have that as follow up PR to this PR? is this kt switch condition is going to block SAVE ANY DID module in edge-agent?
> 
> Can we get this PR merged to unblock ourselves for now.

@sudeshrshetty for now, this PR will support importing signing keys only for ECDSA and ED25519 keys.
Other signing keys will be added later (BBS+, secp256k1)